ci/scripts/execute_build_examples.sh

```diff
@@ -46,7 +46,8 @@ INSTANCE_IP_ADDRESS="$(cat instance-data/instance-ip-address)"
 
 GEODE_VERSION=$(jq -r .semver geode-passing-tokens/*.json)
 
-SET_JAVA_HOME="export JAVA_HOME=/usr/lib/jvm/java-${JAVA_BUILD_VERSION}-openjdk-amd64"
+# SET_JAVA_HOME="export JAVA_HOME=/usr/lib/jvm/java-${JAVA_BUILD_VERSION}-openjdk-amd64"
+SET_JAVA_HOME="export JAVA_HOME=/usr/lib/jvm/bellsoft-java${JAVA_BUILD_VERSION}-amd64"
```

Comment:
Is the lack of a hyphen between `bellsoft-java` and `${JAVA_BUILD_VERSION}` a typo?